tools/serverpod_cli/lib/src/analyzer/entities/validation/restrictions.dart

```diff
@@ -44,20 +46,20 @@ class Restrictions {
     }
 
     var reservedClassNames = {'List', 'Map', 'String', 'DateTime'};
```

Comment:
NIT (non blocking) - Can be made const so that we don't create it at runtime every time we call the method.